Tangem/App/Models/Quotes/QuotesMapper.swift

```diff
@@ -15,12 +15,14 @@ struct QuotesMapper {
                 return nil
             }
 
-            return Quote(
+            let quote = Quote(
                 id: key,
                 price: price,
                 priceChange: value.priceChange24h,
-                prices24h: Array(value.prices24h.values)
+                prices24h: value.prices24h?.values as? [Double]
```

Comment:
```suggestion
                prices24h: value.prices24h.map { Array($0.values) }
```

Так все же надежнее, хоть ошибка вылезет, если тип поменяется

Reply:
Кастить все равно придется из опционала

Reply:
Там в словаре Values , подумаю сейчас

Reply:
я попробовал, у меня ок

Reply:
> я попробовал, у меня ок

Ага смотрю как раз, окэй